src/QsFmt/Parser/QSharpParser.g4

```diff
@@ -21,14 +21,14 @@ namespace
 qualifiedName : Identifier ('.' Identifier)*;
 
 namespaceElement
-    : openDirective # OpenElement
-    | typeDeclaration # TypeElement
+    : directive=openDirective # OpenElement
+    | typeDecl=typeDeclaration # TypeElement
     | callable=callableDeclaration # CallableElement
```

Comment:
```suggestion
    : openDirective # OpenElement
    | typeDeclaration # TypeElement
    | callableDeclaration # CallableElement
```
I suggest we drop the inner labels here, and reorganize the NamespaceItemVisitor a bit. See my comment there.

Reply:
I realize the `callable` label was already there, but now that the other cases are getting filled out, it could be removed.